bibtex-actions.el

```diff
@@ -113,31 +113,19 @@
 
 
 (defcustom bibtex-actions-template
-  '((t . "${author:30}   ${date:8}  ${title:48}"))
+  (cons
+   "${author editor:30}   ${date year issued:8}  ${title:48}"
+   "          ${=key= id:15}    ${=type=:12}    ${tags keywords keywords:*}")
   "Configures formatting for the BibTeX entry.
-When combined with the suffix, the same string is used for
```

Comment:
I guess we should change "BibTeX" to "bibliographic in the line above now.

Reply:
I change all references fof BbTeX to bibliographic, Though the others one might be aspirational as of now.